src/components/pages/search/search.tsx

```diff
@@ -5,13 +5,16 @@ import { observer } from 'mobx-react-lite';
 import ArtefactOverview from '../../structure/visualizing/artefact-overview';
 import SearchSidebar from '../../structure/interacting/search-sidebar';
 import SearchResultNavigation from '../../structure/interacting/search-result-navigation';
+import MyCranach from '../../structure/interacting/my-cranach';
 
 import StoreContext from '../../../store/StoreContext';
 
 import './search.scss';
 
 const Search: FC = () => {
-  const { globalSearch } = useContext(StoreContext);
+  const { globalSearch, ui } = useContext(StoreContext);
+  const isActiveFilter = ui?.sidebar === 'filter' ? 'search__filter--is-active' : '';
```

Comment:
In dieser Datei dann ebenfalls den Enum über den StoreContext importieren:

```ts
import StoreContext, { UISidebarType } from '../../../store/StoreContext';
```

und verwenden:

```ts
const isActiveFilter = ui?.sidebar === UISidebarType.FILTER ? 'search__filter--is-active' : '';
const isActiveMyCranach = ui?.sidebar === UISidebarType.MY_CRANACH ? 'search__my-cranach--is-active' : '';
```